src/action-ros-ci.ts

```diff
@@ -146,13 +147,13 @@ export function validateDistros(
 }
 
 /**
-  * Install ROS dependencies for given packages in the workspace, for all ROS distros being used.
-  */
+ * Install ROS dependencies for given packages in the workspace, for all ROS distros being used.
+ */
 async function installRosdeps(
 	upToPackages: string,
 	workspaceDir: string,
 	ros1Distro?: string,
-	ros2Distro?: string,
+	ros2Distro?: string
```

Comment:
I'd prefer to leave unrelated autoformatting changes out of functional reviews, it makes it hard to effectively review - perhaps if you want to reformat you could introduce that as an enforced linter test in a separate PR to introduce the formatting requirement?

Reply:
These unrelated changes were made by `eslint` or `prettier` through the commit hooks.

I can rebase after #535 is merged.

Reply:
Ok - the formatter is merged - with a rebase and build we should be good to go.

Reply:
rebased and squashed while I was at it!